src/dom/unit-type-conversion.ts

```diff
@@ -93,9 +93,22 @@ const convertChangedValueTypes = (
     const elementStyler = styler(element)
     const originBbox = element.getBoundingClientRect()
     const elementComputedStyle = getComputedStyle(element)
-    const { top, left, bottom, right, transform } = elementComputedStyle
+    const {
+        display,
+        top,
+        left,
+        bottom,
+        right,
+        transform,
+    } = elementComputedStyle
     const originComputedStyle = { top, left, bottom, right, transform }
 
+    // If the element is currently set to display: "none", make it visible before
+    // measuring the target bounding box
+    if (display === "none") {
+        elementStyler.set("display", target.display || "block")
```

Comment:
I assume this behaviour is not really testable?

Reply:
Yeah sadly not. This doesn't result in a flash because we hit multiple renders before the next frame is painted, so it's left in the correct state by then.